webrtc-stats.html

```diff
@@ -538,6 +538,18 @@
           <dt>double availableIncomingBitrate</dt>
           <dd><p>Measured in Bits per second, and is implementation dependent.
             It may be calculated by the underlying congestion control.</p></dd>
+          <dt>double totalRoundTripTime</dt>
+          <dd><p>Represents the total round trip time in ms (for non-retransmitted connectivity checks only).</p></dd>
+          <dt>unsigned long long responses</dt>
+          <dd><p>Total number of connectivity check responses received.</p></dd>
+          <dt>unsigned long long retransmissions</dt>

```

Comment:
The transport behavior of consent checks (https://tools.ietf.org/html/rfc7675) is different from connectivity checks. Consent checks are not re-transmitted (transaction id is changed each time), while connectivity checks re-transmit (with backoff).  Also, you can have "suicide" connectivity checks.